api/HTMLAnchorElement.json

```diff
@@ -445,10 +445,10 @@
               "version_added": "18"
             },
             "firefox": {
-              "version_added": "1"
+              "version_added": "30"
```

Comment:
This is removing claimed support, but http://mdn-bcd-collector.appspot.com/tests/api/HTMLAnchorElement/relList is as simple as a test could be for this, and I've tried it manually on Firefox 29 and 30, this is correct.